src/Microsoft.TestPlatform.CommunicationUtilities/Serialization/TestResultConverter.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.CommunicationUtilities.Serialization
+{
+    using System;
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+    using Newtonsoft.Json;
+
+    /// <inheritdoc/>
+    public class TestResultConverter : JsonConverter
```

Comment:
Require documentation. Should we include these classes in versioned folders (same for TestCaseConverter)?